src/main/java/org/isf/admission/manager/AdmissionBrowserManager.java

```diff
@@ -183,7 +183,7 @@ public int newAdmissionReturnKey(Admission admission) throws OHServiceException
 	 * @throws OHServiceException
 	 */
 	public boolean updateAdmission(Admission admission) throws OHServiceException {
-		//validateAdmission(admission, false);
+		validateAdmission(admission, false);
```

Comment:
Why to comment out the validation method?

Reply:
you can remove the comment.

Reply:
I did ;)